src/main/java/nl/tudelft/contextproject/gui/PresetController.java

```diff
@@ -1,34 +1,116 @@
 package main.java.nl.tudelft.contextproject.gui;
 
+import java.io.IOException;
```

Comment:
CheckStyle gives errors about these imports, saying they are not in the wrong order.